pkg/apis/registryman.kubermatic.com/v1alpha1/testdata/test_options/global-registry-forcedelete.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: registryman.kubermatic.com/v1alpha1
+kind: Registry
+metadata:
+  name: global
+spec:
+  provider: harbor
+  role: GlobalHub
+  apiEndpoint: http://core.harbor-1.demo
+  username: admin
+  password: admin
+  options:
+    forceDelete: false
```

Comment:
An idea: what if you used annotations for the options rather than adding a new field?
```yaml
metadata:
  name: global
  annotations:
    registryman.kubermatic.com/forcedelete: true
```

Reply:
done